### PR TITLE
20454 - PUB/SUB Integration TEST on Python 3.8

### PR DESCRIPTION
**Pull request by seeker25**
https://github.com/bcgov/entity/issues/20454

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: pay-api/src/pay_api/services/payment_account.py [hunk 1]
```diff
@@ -831,29 +831,21 @@ def create_account_event_payload(self, event_type: str, receipt_info: dict = Non
...
-            'type': event_type,
-            'source': f'https://api.pay.bcregistry.gov.bc.ca/v1/accounts/{self.auth_account_id}',
-            'id': f'{self.auth_account_id}',
-            'time': f'{datetime.now()}',
-            'datacontenttype': 'application/json',
-            'data': {
-                'accountId': self.auth_account_id,
-                'accountName': self.name
-            }
+            'accountId': self.auth_account_id,
```
Comment (seeker25) [inline pay-api/src/pay_api/services/payment_account.py:834]: `QueueMessageTypes.PAD_ACCOUNT_CREATE.value`
`QueueMessageTypes.NSF_UNLOCK_ACCOUNT.value`
`EFT_AVAILABLE_NOTIFICATION`

need to be tested

Comment (seeker25) [reply pay-api/src/pay_api/services/payment_account.py:834]: PAD_ACCOUNT_CREATE works

Comment (seeker25) [reply pay-api/src/pay_api/services/payment_account.py:834]: EFT_AVAILABLE works

**Comment by seeker25** [comment]
Mostly going through payment-jobs now, making sure all that pub stuff is working.

**Review by ochiu — APPROVED** [review]
LGTM, just the minor linting error

**Review by Jxio — APPROVED** [review]

**Pull request merged**